keyboards/abko/ak84bt/info.json

```diff
@@ -123,8 +123,8 @@
     },
     "usb": {
         "device_version": "0.0.1",
-        "pid": "0x4321",
-        "vid": "0x7654"
+        "pid": "0xC821",
+        "vid": "0x0483"
```

Comment:
This appears to be STMicroelectron's VID, with an unused PID.  
https://www.the-sz.com/products/usbid/index.php?v=0x0483&p=&n=

Do you have explicit permission from STM to use it?

Reply:
Sorry for the for the misunderstanding in the previous information. We have applied to the USB-IF Administration to join and obtained the corresponding VID. The updated VID PID is based on the VID filled in upon application.